scripts/release.ts

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/env -S npx ts-node -P tsconfig.scripts.json
+
+import { execSync } from "child_process";
+
+const SV_COMMAND = "yarn run standard-version --infile ./CHANGELOG.md";
+
+const verboseExec = (cmd: string) => {
+  let result;
+
+  try {
+    result = execSync(cmd, { stdio: "pipe" }).toString("utf-8");
```

Comment:
Can we use `stdio: ['ignore', 'pipe', 'inherit']`? This will prevent piping stdin to the command and will print stderr which is useful for debugging, even if the command does not throw..

Reply:
My thinking was that this check is non-verbose, to not muddy the already long output with it.
If we want it to be more verbose, I could also just use `verboseExec` here and remove some duplication while at it.

Reply:
04e0c0ec4d6dba64316dc167fcb7ba403e004cbd

Reply:
Now that I tried it out, I changed my mind, the git status output interferes with our logging:

```
  ✔ git branch release-v1.0.0 && git checkout release-v1.0.0
  ✔ standard-version --infile ./CHANGELOG.md --silent --sign --release-as major
Enumerating objects: 7, done.
Counting objects: 100% (7/7), done.
Delta compression using up to 8 threads
Compressing objects: 100% (4/4), done.
Writing objects: 100% (4/4), 1.35 KiB | 1.35 MiB/s, done.
Total 4 (delta 3), reused 0 (delta 0)
remote: Resolving deltas: 100% (3/3), completed with 3 local objects.
remote:
remote: Create a pull request for 'release-v1.0.0' on GitHub by visiting:
remote:      https://github.com/rudolfs/radicle-upstream/pull/new/release-v1.0.0
remote:
To github.com:rudolfs/radicle-upstream.git
 * [new branch]        release-v1.0.0 -> release-v1.0.0
  ✔ git push origin release-v1.0.0
Everything up-to-date
  ✔ hub pull-request -p --no-edit
```

0d2914450c01cc7be3091273ce2b36b3f5fcb691